tests/topotests/bgp-route-map/test_route_map_topo2.py

```diff
@@ -1013,18 +1014,18 @@ def test_modify_prefix_list_referenced_by_rmap_p0():
     protocol = 'bgp'
     for addr_type in ADDR_TYPES:
         result = verify_rib(tgen, addr_type, dut, input_dict,
-                protocol=protocol)
+                protocol=protocol, expected=False)
         assert result is not True, 'Testcase {} : Failed \n'
-        'Expected behaviour: routes are not present \n '
-        'Error: {}'.format(
+        'routes are not present \n Error: {}'.format(
```

Comment:
nit: please use double quotes or single quotes everywhere, to keep code clean and consistent.

Reply:
Done :)